try.html

```diff
@@ -124,6 +124,10 @@ <h3 id="build"> Build </h3>
       <td><img src='/shippable/54d119db5ab6cc13528ab183/master.svg' alt=''/></td>
       <td><code>https://img.shields.io/shippable/54d119db5ab6cc13528ab183/master.svg</code></td>
   </tr>
+  <tr><th> Snap CI branch: </th>
+      <td><img src='/snap-ci/snap-ci/snap-deploy/master.svg' alt=''/></td>
+      <td><code>https://img.shields.io/snap-ci/snap-ci/snap-deploy/master.svg</code></td>
```

Comment:
So `snap-ci` is the username, and `snap-deploy` is the repo name? I think having a double `snap-ci` makes the example less readable.

Could we show `https://img.shields.io/snap-ci/USER/REPO/BRANCH.svg` maybe? Or use a more obvious example?